src/centos/scripts/start-marklogic.sh

```diff
@@ -401,10 +445,11 @@ if [[ "${MARKLOGIC_VERSION}" =~ "10" ]] || [[ "${MARKLOGIC_VERSION}" =~ "9" ]];
 else 
      HEALTH_CHECK="7997/LATEST/healthcheck"
 fi
+ML_HOST_PROTOCOL=$(get_host_protocol "localhost" "7997")
 
 while true
 do
-    HOST_RESP_CODE=$(curl http://"${HOSTNAME}":"${HEALTH_CHECK}" -X GET -o host_health.xml -s -w "%{http_code}\n")
+    HOST_RESP_CODE=$(curl "${ML_HOST_PROTOCOL}"://localhost:"${HEALTH_CHECK}" -X GET -o host_health.xml -s -w "%{http_code}\n" -k)
```

Comment:
Why using -k as the user is supposed to provide the CA Cert ?

Reply:
update code to use CA cert